packages/components/src/checkbox-control/README.md

```diff
@@ -56,17 +56,16 @@ If only a few child checkboxes are checked, the parent checkbox becomes a mixed
 Render an is author checkbox:
 ```jsx
 import { CheckboxControl } from '@wordpress/components';
-import { withState } from '@wordpress/compose';
+import { useState } from '@wordpress/element';
 
-const MyCheckboxControl = withState( {
-	isChecked: true,
-} )( ( { isChecked, setState } ) => (
+const MyCheckboxControl = () => (
+	const [ checked, setChecked ] = useState( true );
```

Comment:
To align with the usage it needs to be:
```suggestion
	const [ isChecked, setChecked ] = useState( true );
```

Reply:
The usage in our code? In that case `setChecked` should be `setIsChecked`.